cluster/plan.go

```diff
@@ -52,13 +52,15 @@ const (
 	KubeletDockerConfigPath    = "/var/lib/kubelet/config.json"
 
 	// MaxEtcdOldEnvVersion The versions are maxed out for minor versions because -rancher1 suffix will cause semver to think its older, example: v1.15.0 > v1.15.0-rancher1
-	MaxEtcdOldEnvVersion      = "v3.2.99"
-	MaxK8s115Version          = "v1.15"
-	MaxEtcdPort4001Version    = "v3.4.3-rancher99"
-	MaxEtcdNoStrictTLSVersion = "v3.4.14-rancher99"
-	MaxK8s121Version          = "v1.21.99-rancher99"
-	MaxK8s122Version          = "v1.22.99-rancher99"
-
+	MaxEtcdOldEnvVersion             = "v3.2.99"
+	MaxK8s115Version                 = "v1.15"
+	MaxEtcdPort4001Version           = "v3.4.3-rancher99"
+	MaxEtcdNoStrictTLSVersion        = "v3.4.14-rancher99"
+	MaxK8s121Version                 = "v1.21.99-rancher99"
+	MaxK8s122Version                 = "v1.22.99-rancher99"
+	JanK8s123Version                 = "v1.23.16-rancher1"
+	AprilK8s124Version               = "v1.24.13-rancher1"
+	AprilK8s125Version               = "1.25.9-rancher1"
```

Comment:
For consistency we want to add `v` here as well.
```suggestion
	AprilK8s125Version               = "v1.25.9-rancher1"
```

Reply:
Rectified the incorrect version.